generate.py

```diff
@@ -133,10 +137,12 @@ def main(
     encoded = tokenizer.encode(prompt, device=fabric.device)
     prompt_length = encoded.size(0)
 
+    T_new = encoded.size(0) + max_new_tokens
```

Comment:
We should add back here the code removed in https://github.com/Lightning-AI/lit-parrot/pull/67/files